tox.ini

```
@@ -11,3 +11,11 @@ deps =
 
 [flake8]
 ignore = W504, E501
+
+[testenv:mypy]
+basepython = python3
+commands =
+    mypy src/josepy
+deps:
+    # installs the test dependencies as specified in setup.py
+    .[dev]
```

nit: to be consistent with the existing style of this file.

```suggestion
deps =
    -e .[dev]
```